.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -3,6 +3,13 @@ description: Suggest an idea for this project not related to bugs
 title: "[Feature request] Feature request title"
 labels: ["enhancement"]
 body:
+  - type: textarea
+    id: motivation
+    attributes:
+      label: Motivation
+      description: |
+        Is your feature request related to a problem? Please describe. 
+        Give clear and concise description of what the problem is. Ex. I am always frustrated when (...).
```

If we are moving it up here we must force the user to justify the change.
```suggestion
  - type: textarea
    id: motivation
    attributes:
      label: Motivation
      description: |
        Is your feature request related to a problem? Please describe. 
        Give clear and concise description of what the problem is. Ex. I am always frustrated when (...).
    validations:
      required: true
```

---

Requesting users to describe the issue will allow us to evaluate the proposed changes and also help us formulate our own proposals.

---

It is difficult to prioritize or offer alternatives if we do not know why the user is making the request. My opinion is that the motivation statement is more important than the proposal.